pkgs/development/libraries/physics/geant4/datasets.nix

```diff
@@ -1,22 +1,32 @@
 { stdenv, fetchurl, }:
```

Comment:
Need to add G4TENDL1.3.2

Reply:
Strangely enough I could not find the corresponding environment variable we would have to set for it. Perhaps we can skip this data for now.

Reply:
I couldn't find it either, which is why I skipped it.  I grepped through the geant4 code as well, and I don't know if it's even used?

Reply:
Let's assume it is not needed.